change/@ni-nimble-components-a29fed7e-5696-4a11-bb75-66245de27363.json

```
@@ -0,0 +1,7 @@
+{
+  "type": "minor",
```

For "Incubating" components we've established a policy that all changes can be marked patch. While this is not technically aligned with semantic versioning since they are introducing new APIs, we're saying that those APIs are still not "public" so don't need to be advertized to clients via a minor bump.

I'll just go ahead and apply this change.

```suggestion
  "type": "patch",
```

---

Thanks for updating.